backend/barista_framework.md

```diff
@@ -0,0 +1,11 @@
+### Barista Framework
+
+There is a set of components that are spread and/or duplicated across several projects, but can be tied up as a `clojars` package in order to be reused seamless on all API's.
+
+- CORS ruler
```

Comment:
Haha I was also unsure about the name too, maybe `CORS Middleware` to be more totally specific, wdyt?